app/src/main/java/io/github/droidkaigi/confsched2018/util/ext/BottomNavigationViewHelper.kt

```diff
@@ -0,0 +1,34 @@
+package io.github.droidkaigi.confsched2018.util.ext
+
+import android.annotation.SuppressLint
+import android.support.design.internal.BottomNavigationItemView
+import android.support.design.internal.BottomNavigationMenuView
+import android.support.design.widget.BottomNavigationView
+import android.util.Log
+
+internal object BottomNavigationViewHelper {
+
+    @SuppressLint("RestrictedApi")
+    fun disableShiftMode(view: BottomNavigationView) {
```

Comment:
I think it's better to implement by using Extention. like `util/ext/BottomNavigationViewExt.kt`.
What do you think?

Reply:
It's supposed to be. 
I'm not sure if this is approval, but I'll fix this. 
Thx for your good feedback.